py/torch_tensorrt/dynamo/conversion/impl/reduce.py

```diff
@@ -208,3 +209,25 @@ def mean(
     )
     set_layer_name(layer, target, name, source_ir)
     return layer.get_output(0)
+
+
+def any(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input_val: TRTTensor,
+    dim: Sequence[int] = [],
```

Comment:
Based on the supported schemas, specifically [`any.dim` and `any.dims`](https://github.com/pytorch/pytorch/blob/534c73d478fc967cb3a6a29f5eada94bc4ce2c29/aten/src/ATen/native/native_functions.yaml#L721-L727), this would need to be:
```python
dim: Union[int, Optional[Sequence[int]]] = None,
```